sdk/js/src/Client.ts

```diff
@@ -168,25 +172,26 @@ export class DVCClient implements Client {
                 this.eventQueue.flushEvents()
                 const oldConfig = this.config || {} as BucketedUserConfig
 
-                getConfigJson(this.environmentKey, anonUser)
-                    .then((config) => {
-                        this.config = config as BucketedUserConfig
-                        this.user = anonUser
-                        this.store.saveConfig(config).then(() => {
-                            console.log('Successfully saved config to local storage')
-                        })
-                        this.store.saveUser(anonUser).then(() => {
-                            console.log('Successfully saved user to local storage!')
-                        })
-                        const oldFeatures = oldConfig.features || {}
-                        const oldVariables = oldConfig.variables || {}
-                        this.eventEmitter.emitFeatureUpdates(oldFeatures, config.features)
-                        this.eventEmitter.emitVariableUpdates(oldVariables,
-                            config.variables, this.variableDefaultMap)
-                        resolve(config.variables || {})
-                    }).catch((e) => {
-                        reject(e)
+                this.requestConsolidator.queue('identify',
```

Comment:
Should it be 'reset' here?

Reply:
they're both an `identify` type action, and we don't want them independently being called as they save the config; we want them to be consistent.

i.e. calling `identifyUser`, then `resetUser` will always give us the bucketing for the anonymous user from the result of `resetUser` for both promises.